corehq/warehouse/models/staging.py

```diff
@@ -115,16 +134,22 @@ def field_mapping(cls):
         ]
 
     @classmethod
-    def raw_record_iter(cls, start_datetime, end_datetime):
+    def record_iter(cls, start_datetime, end_datetime):
         user_ids = get_user_ids_by_last_modified(start_datetime, end_datetime)
 
         return iter_docs(CouchUser.get_db(), user_ids)
 
 
-class DomainStagingTable(StagingTable):
-    slug = 'domain_staging'
+class DomainStagingTable(StagingTable, CouchToDjangoETLMixin):
+    '''
+    Represents the staging table to dump data before loading into the DomainDim
+
+    Grain: domain_id
+    '''
+    slug = DOMAIN_STAGING_SLUG
 
     domain_id = models.CharField(max_length=255)
+    domain = models.CharField(max_length=100)
```

Comment:
it could be, but since every dimension has `domain` on it, it'll be converted to `domain` anyways: https://github.com/dimagi/commcare-hq/blob/br/prep-appstatus/corehq/warehouse/models/dimensions.py#L23. thought it'd be simpler to have the domain be named the same thing on every model, even if it's the domain dim